docs/simulate.ron

```diff
@@ -249,6 +249,25 @@
             /* sigma for the Gaussian dispersal kernel N(0, sigma^2) */
             sigma: (0.0 <= f64),
         )
+        /* (almost) infinite spatially-explicit scenario with Clark 2Dt distributed dispersal
+         * the entire infinite landscape is habitat but, without loss of generality, has deme 1
+         * the landscape is on a (wrapping) torus with 0 <= x < 2^32 and 0 <= y < 2^32 */
+      | Clark(
+            /* rectangular sample area, individuals living in here are simulated
+             * the sample area can wrap around the torus */
+            sample: Extent(
+                /* lower-left origin x of the sample area */
+                x: (u32),
```

Comment:
point 4 from the email. If the landscape is wrapped, does it matter where we put the lower left corner of the sample area?

Reply:
No(*). In the general case, the corner is just an offset and so you can choose one that you prefer - e.g. if you choose (-width/2, -height/2) as (u32, u32) then you can later reinterpret all coordinates as signed integers again and everything is nicely centred around zero. As long as individuals don’t disperse to the left and come out on the right and meet other individuals you’re also simulating (for this they would need to traverse a distance of 2^32), the wrapping has no impact on the results.

(*) The independent algorithm seeds its randomness based on the coordinates, so changing the origin will result in a different result drawn from the same result probability distribution (the same effect as changing the global seed).

Reply:
Ahh ok good. I just responded to your email explaining that I changed the corner to (1,1) to make it easier read the results in to a matrix (for Julia and R because of the indexing).